### PR TITLE
fix(platform): fix displaying navigation buttons in wizard generator

### LINKED ISSUE #6007: Bug: Wizard generator in embeded mode with appendToWizard=false does not show navigation buttons
#### Is this a bug, enhancement, or feature request?
Bug
#### Briefly describe your proposal.
In embedded wizard generator mode (not dialog one), if user set's `[appendToWizard]="false"`, navigation buttons are not showing.
Additionally, example for custom navigation buttons for embedded wizard is missing.

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
v0.31.0
#### If this is a bug, please provide steps for reproducing it.
1. In first example of Wizard generator, add  `[appendToWizard]="false"`
ER: "Go next" button should be visible
AR: "Go next" button is hidden

### PR DESCRIPTION
**Pull request by N1XUS**
#### Please provide a link to the associated issue.
Closes #6007 

#### Please provide a brief summary of this pull request.
1. Added additional input for wizard body to render navigation button when `appendToWizard` is `false`.
2. Added one more example of customizable wizard generator when in embedded mode
#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- n/a update `README.md`
- n/a [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: apps/docs/src/app/platform/component-docs/platform-wizard-generator/platform-wizard-generator-docs.component.html [hunk 1]
```diff
@@ -39,10 +39,24 @@
     <br><br>
     Buttons appearance can be changed with <code>[navigationButtonLabels]</code> property.
     <br><br>
+    If you want to use completely different style for navigational buttons,
+    you can use special directives: <code>fdpWizardGeneratorGoNextButton, fdpWizardGeneratorFinishButton</code>.
+</description>
+<component-example>
+    <fdp-wizard-generator-customizable-embeded-example></fdp-wizard-generator-customizable-embeded-example>
+</component-example>
+<code-example [exampleFiles]="customizableExample"></code-example>
```
Comment (platon-rov) [inline apps/docs/src/app/platform/component-docs/platform-wizard-generator/platform-wizard-generator-docs.component.html:48]: I guess there should be different example files as `customizableExample` appears twice in docs whether different components are shown.

**Review by InnaAtanasova — APPROVED** [review]

File: apps/docs/src/app/platform/component-docs/platform-wizard-generator/platform-wizard-generator-docs.component.html (hunk 1 above)
Comment (N1XUS) [reply apps/docs/src/app/platform/component-docs/platform-wizard-generator/platform-wizard-generator-docs.component.html]: @platon-rov please check.

**Review by platon-rov — APPROVED** [review]

**Review by droshev — CHANGES_REQUESTED** [review]
@N1XUS it could be just a doc change but the wizard in a dialog(core) has better paddings/margins and the buttons' footer is according to the specs, but the wizard in a dialog(generator) is not
Screen Shot 2021-07-29 at 9 29 05 PM
Screen Shot 2021-07-29 at 9 28 39 PM

**Comment by N1XUS** [comment]
@droshev Wizard in core has custom styling for example components.
All paddings for wizard generator were taken from fundamental-styles specs for the wizard.

**Comment by mikerodonnell89** [comment]
> @droshev Wizard in core has custom styling for example components.
> All paddings for wizard generator were taken from fundamental-styles specs for the wizard.

@droshev @N1XUS Previously I was using custom styles in the docs to add paddings to the core wizard, but recently updated it so that it uses the `size` inputs on the wizard progress bar and the wizard content components. Perhaps the platform wizard can do something similar, maybe handling it automatically on screen sizes?  But this should not be considered part of this issue so it'll be a separate PR

**Review by Betrozov — APPROVED** [review]

**Review by droshev — APPROVED** [review]

**Pull request merged**